libkbfs/mdserver_remote.go

```diff
@@ -351,6 +354,8 @@ func (md *MDServerRemote) OnDisconnected(ctx context.Context,
 		md.log.CWarningf(ctx, "MDServerRemote is disconnected")
 		md.config.Reporter().Notify(ctx,
 			connectionNotification(connectionStatusDisconnected))
+		md.reconnectBackoff.Start(reconnectBackoffWindow)
+		defer md.reconnectBackoff.Wait()
```

Comment:
That was the only approach I could think of without changing the interface. But it seems to be a good idea to just change the interface now. Hope there won't be too much shave!